frontend/src/metabase/query_builder/components/DataSelector/DataSelectorTablePicker/DataSelectorTablePicker.unit.spec.tsx

```diff
@@ -1,43 +1,67 @@
 import { checkNotNull } from "metabase/core/utils/types";
 import { getMetadata } from "metabase/selectors/metadata";
-import { SchemaId, TableId } from "metabase-types/api";
+import { InitialSyncStatus, Schema, Table } from "metabase-types/api";
```

Comment:
nit: if it's typing we should try to use `import type`, unless it doesn't work. There's been some reports that using `import type` doesn't work in some circumstances.

```suggestion
import type { InitialSyncStatus, Schema, Table } from "metabase-types/api";
```